src/components/shared/Entity/Details/RelatedEntities/index.tsx

```diff
@@ -0,0 +1,93 @@
+import type { RelatedEntitiesProps } from 'src/components/shared/Entity/Details/RelatedEntities/types';
+
+import { useMemo } from 'react';
+
+import { Skeleton } from '@mui/material';
+
+import { useIntl } from 'react-intl';
+
+import ChipList from 'src/components/shared/ChipList';
+import useDetailsNavigator from 'src/hooks/useDetailsNavigator';
+import { useLiveSpecs_parentCapture } from 'src/hooks/useLiveSpecs';
+import { ENTITY_SETTINGS } from 'src/settings/entity';
+
+function RelatedEntities({
+    collectionId,
+    entityType,
+    newWindow,
+    preferredList,
+}: RelatedEntitiesProps) {
+    const intl = useIntl();
+
+    const { generatePath } = useDetailsNavigator(
+        ENTITY_SETTINGS[entityType].routes.details
+    );
+
+    // If we have a list then just send null over and never run a query
+    const { data, error, isValidating } = useLiveSpecs_parentCapture(
+        preferredList ? null : (collectionId ?? null),
+        entityType
+    );
+
+    const dataToShow = useMemo(() => {
+        if (preferredList) {
+            return preferredList;
+        } else {
+            return data?.map((datum) => datum.live_specs.catalog_name);
+        }
+    }, [data, preferredList]);
```

Comment:
New

We now can go fetch live_specs_flow related data if we are not provided a list before hand.